modules/etl/talend/talend-definition/src/main/java/com/liferay/talend/tliferayoutput/Action.java

```diff
@@ -19,7 +19,7 @@
  */
 public enum Action {
 
-	DELETE("DELETE"), INSERT("POST"), UPDATE("PUT");
+	Delete("DELETE"), Update("PUT"), Upsert("POST");
```

Comment:
Why this? AFAIK Liferay always capitalizes enums.

Reply:
Because the Enums will be used on the UI, this is how the Enumeration Widget works in Talend Studio and it's more convenient for the eye, IMHO, see 
![enumwidget](https://user-images.githubusercontent.com/2045409/41302606-301667b6-6e6b-11e8-9d34-06e3c5708d2b.jpg)